zendesk_tickets/client.py

```diff
@@ -0,0 +1,35 @@
+import json
+
+import requests
+from django.conf import settings
+
+
+TICKETS_URL = settings.ZENDESK_BASE_URL + '/api/v2/tickets.json'
```

Comment:
I really wish more APIs that claim to be RESTful actually were, so we didn't have to do things like this :cry: